### ISSUE TITLE
Tree ARIA role group confusion

### ISSUE THREAD

**Issue by FishOfPrey** [salesforce-ux/design-system#351]
In the documentation for [Trees](https://www.lightningdesignsystem.com/components/trees/) it states:

> Whenever the tree has a nested group, the li element should receive the ARIA role `group`. This will add appropiate styling to decendent .tree__item elements. 

Yet in the example the **group** role is applied to the ul element instead and the li gets a **treeitem** role.

```html
<ul class="slds-is-collapsed" role="group" aria-labelledby="tree0-node1__label">
        <li id="tree0-node1-0" role="treeitem" aria-level="2">
```

Which is the correct approach?

**Comment by kaelig** [comment]
Thank you for your feedback and for using the Lightning Design System, @FishOfPrey!

@SiTaggart that looks like a documentation bug, would you agree?

**Comment by SiTaggart** [comment]
Correct, the documentation is out of date. The markup tells the better story.

Sorry for the confusion!

**Comment by FishOfPrey** [comment]
Thanks, I'll stick with the markup examples.

**Comment by engai** [comment]
@FishOfPrey The Tree documentation has been updated since and should have the correct information. https://www.lightningdesignsystem.com/components/trees/

Closing this, but feel free to reopen if needed. Thanks.
